pkg/trace/traceutil/otel_util.go

```diff
@@ -167,7 +168,7 @@ func GetOTelService(span ptrace.Span, res pcommon.Resource, normalize bool) stri
 func GetOTelResource(span ptrace.Span, res pcommon.Resource) (resName string) {
 	resName = GetOTelAttrValInResAndSpanAttrs(span, res, false, "resource.name")
 	if resName == "" {
-		if m := GetOTelAttrValInResAndSpanAttrs(span, res, false, semconv.AttributeHTTPMethod); m != "" {
+		if m := GetOTelAttrValInResAndSpanAttrs(span, res, false, "http.request.method", semconv.AttributeHTTPMethod); m != "" {
```

Comment:
are there semconv constants that can be used for these strings here? (also line 268)

Reply:
Yes, changed to const instead.